locmai/humble
# Init Nginx Ingress

```hcl
Terraform will perform the following actions:

  # helm_release.nginx-ingress will be created
  + resource "helm_release" "nginx-ingress" {
      + atomic                     = false
      + chart                      = "nginx-ingress"
      + cleanup_on_fail            = false
      + create_namespace           = true
      + dependency_update          = false
      + disable_crd_hooks          = false
      + disable_openapi_validation = false
      + disable_webhooks           = false
      + force_update               = false
      + id                         = (known after apply)
      + lint                       = false
      + max_history                = 0
      + metadata                   = (known after apply)
      + name                       = "nginx"
      + namespace                  = "nginx"
      + recreate_pods              = false
      + render_subchart_notes      = true
      + replace                    = false
      + repository                 = "https://helm.nginx.com/stable"
      + reset_values               = false
      + reuse_values               = false
      + skip_crds                  = false
      + status                     = "deployed"
      + timeout                    = 300
      + values                     = [
          + <<-EOT
                controller:
                    service:
                        annotations:
                            metallb.universe.tf/address-pool: default-pool
            EOT,
        ]
      + verify                     = false
      + version                    = "0.8.0"
      + wait                       = true
    }

Plan: 1 to add, 0 to change, 0 to destroy.
```
